.github/workflows/go.yml

```diff
@@ -19,10 +19,23 @@ jobs:
         go-version-file: "go.mod"
 
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@v3
+      uses: smartcontractkit/golangci-lint-action@54ab6c5f11d66a92d14c3f7cc41ea13f676644bd
```

Comment:
What's the significance of this particular version?

Reply:
This is the version used in the main chainlink repo as well. 
https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/ci-core.yml#L82-L85
This change is to support multiple --out-format args